autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -0,0 +1,757 @@
+                          msg='The create failed with {0} for group '
+                          '{1}'.format(pool_group_resp.status_code, pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].loadBalancerId,
+                          self.pool_1.id,
+                          msg='The launchConfig for group {0} did not contain the load balancer'
+                          .format(pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].type, "RackConnectV3",
+                          msg='Load balancer type {0} is not correct for RackConnect pools'
+                          .format(pool_group.launchConfiguration.loadBalancers[0].type))
+
+    #@unittest.skip("Skipping")
+    @tags(speed='slow', type='rcv3')
+    def test_create_scaling_group_with_pool_on_public(self):
+        """
+        Test that it is possible to create a scaling group with 0 entities
+        connected to an RCV3 LB pool with only private network specified
+        """
+        # Create a scaling group with zero servers
+        lb_pools = [{'loadBalancerId': self.pool_1.id, 'type': 'RackConnectV3'}]
+        pool_group_resp = self._create_rcv3_group(lb_list=lb_pools,
+                                                  group_min=0,
+                                                  network_list=[self.public_network])
+
+        pool_group = pool_group_resp.entity
+
+        self.assertTrue(pool_group_resp.ok,
+                        msg='Create scaling group call failed with API Response: {0} for '
+                        'group {1}'.format(pool_group_resp.content, pool_group.id))
+        self.assertEquals(self.create_resp.status_code, 201,
+                          msg='The create failed with {0} for group '
+                          '{1}'.format(pool_group_resp.status_code, pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].loadBalancerId,
+                          self.pool_1.id,
+                          msg='The launchConfig for group {0} did not contain the load balancer'
+                          .format(pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].type, "RackConnectV3",
+                          msg='Load balancer type {0} is not correct for RackConnect pools'
+                          .format(pool_group.launchConfiguration.loadBalancers[0].type))
+
+    #@unittest.skip("Skipping")
+    @tags(speed='slow', type='rcv3')
+    def test_create_scaling_group_with_pool_counts(self):
+        """
+        Test that it is possible to create a scaling group with min_servers servers
+        connected to an RCv3 LB pool.
+
+        This is a simple smoke test that only checks the node counts.
+        """
+        # This sleep is necessary because other tests have just completed, but
+        # their resources haven't yet been completely freed up.  TODO(sfalvo):
+        # See Github issue #855.
```

Comment:
Is that cause delete scaling group is taking long or failing in unknown ways. 

I am not sure I understand how polling on the delete scaling group will help us fix the issue (#855) of disposing all the resources from the previous test, cause receiving a 204 on a delete group does not mean that the resources in it have been deleted successfully. I still think performing the empty scaling group followed by polling to see that the actually group empties in the tearDown would help.